--- a/hw/mips/Kconfig
+++ b/hw/mips/Kconfig
@@ -13,6 +13,11 @@
     select SERIAL_MM
     select SMBUS_EEPROM
 
+config SF2000
+    bool
+    default y
+    depends on MIPS && !TARGET_BIG_ENDIAN
+
 config JAZZ
     bool
     default y
--- a/hw/mips/meson.build
+++ b/hw/mips/meson.build
@@ -4,6 +4,7 @@
 common_ss.add(when: 'CONFIG_LOONGSON3V', if_true: files('loongson3_bootp.c'))
 mips_ss.add(when: 'CONFIG_LOONGSON3V', if_true: files('loongson3_virt.c'))
 mips_ss.add(when: 'CONFIG_MALTA', if_true: files('malta.c'))
+mips_ss.add(when: 'CONFIG_SF2000', if_true: files('sf2000.c'))
 mips_ss.add(when: 'CONFIG_MIPS_CPS', if_true: files('cps.c'))
 
 if 'CONFIG_TCG' in config_all_accel
